src/changelog/.2.x.x/2778_configurator_no_op.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<entry xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xmlns="https://logging.apache.org/xml/ns"
+       xsi:schemaLocation="https://logging.apache.org/xml/ns https://logging.apache.org/xml/ns/log4j-changelog-0.xsd"
+       type="fixed">
```

Comment:
Should be `changed`, since it changes the previous behavior.

Reply:
@ppkarwasz, maybe indeed `changed` is a better option to stress the change in the behaviour.

Reply:
I agree that 'changed' is better.

Reply:
Fixed in https://github.com/apache/logging-log4j2/pull/2783/commits/e6c7f197f2f9a3dda109b669ea9b2d73d5a77d24